### DISCUSSION TITLE
Any easy way to have both the killclick and hitclick script?

### DISCUSSION THREAD

**Discussion by o777l** [winstxnhdw/lc-hax#387] [General]
I liked using hitclick to easily obtain a shotgun from the nutcracker but now he's deleted, including the gun.

I have a rough idea of how it works to add but I'm not an actual coder mind you

**Comment by winstxnhdw** [comment]
Oh dear, `killclick` should still preserve the shotgun. Let me take a look at it.

**Comment by winstxnhdw** [comment]
Can you try it again? I just pushed a fix for it.

**Comment by o777l** [comment]
> Can you try it again? I just pushed a fix for it.

Tried cloning again and still removes the shotgun. I noticed while using ESP, that not only does it delete the enemy in crosshair, but every other enemy withiin a very big radius. Even deletes through walls?

Edit: Only nutcracker was completely deleted, the other enemies drop dead

**Reply by winstxnhdw** [reply]
Finally, got it to work. I can even make it spawn bullets.

![image](https://github.com/winstxnhdw/lc-hax/assets/56998716/9854085f-d619-4e79-971e-295634bc1365)
